web/src/components/public/resourceAllocation/Allocation.js

```diff
@@ -15,26 +15,26 @@ const allocationMap = [
   {
```

Comment:
`web/src/components/public/resourceAllocation/Allocation.js`

In backend the term is `quota`, not `allocation`.